app/controllers/admin/quizzs_controller.rb

```diff
@@ -0,0 +1,116 @@
+class Admin::QuizzsController < ApplicationController
+
+  before_action :authenticate_user!
```

Comment:
you need to create Admin::Admin controller, and all controllers in admin namespace should inherit from it, then you can remove before_action :authenticate_user! from all, and maybe other actions too